modules/core/src/hal_replacement.hpp

```diff
@@ -860,19 +860,19 @@ inline int hal_ni_gemm64fc(const double* src1, size_t src1_step, const double* s
 
 /**
    @brief Finds the global minimum and maximum in an array.
-   @param src_data Source image
-   @param src_step Source image
-   @param width Source image dimensions
-   @param height Source image dimensions
+   @param src_data Source image data
+   @param src_step Source image step between rows
+   @param width Source image width
+   @param height Source image height
    @param depth Depth of source image
-   @param minVal Pointer to the returned global minimum and maximum in an array.
-   @param maxVal Pointer to the returned global minimum and maximum in an array.
-   @param minIdx Pointer to the returned minimum and maximum location.
-   @param maxIdx Pointer to the returned minimum and maximum location.
-   @param mask Specified array region.
+   @param minVal Pointer to the returned global minimum value in the array
+   @param maxVal Pointer to the returned global maximum value in the array
+   @param minOffset Pointer to the returned minimum offset in the array
+   @param maxOffset Pointer to the returned maximum offset in the array
+   @param mask_data Mask data pointer
 */
 inline int hal_ni_minMaxIdx(const uchar* src_data, size_t src_step, int width, int height, int depth, double* minVal, double* maxVal,
-                            int* minIdx, int* maxIdx, uchar* mask) { return CV_HAL_ERROR_NOT_IMPLEMENTED; }
```

Comment:
Perhaps we should add a new interface instead of trying to fix/change existed due to compatibility reasons.

Reply:
Makes sense, added it as `minMaxOffset` along with existing one

Reply:
I'm just not sure what to do with existing HAL which is not called anymore.
This can be discussed; I'm ready to roll the previous code back.